Content.Server/Anomaly/Effects/InnerBodyAnomalySystem.cs

```diff
@@ -186,6 +187,9 @@ private void OnMobStateChanged(Entity<InnerBodyAnomalyComponent> ent, ref MobSta
         if (args.NewMobState != MobState.Dead)
             return;
 
+        if (EntityManager.HasComponent<PendingZombieComponent>(args.Target))
```

Comment:
You should use `HasComp<PendingZombieComponent>(args.Target)` instead (Same thing but neater)! Also, what you should probably also do is instead send an event the target seeing if it wants to cancel, and then then the zombie system can listen for that and then cancel it. If you need some examples of how this is done let me know or ask in the discord!

Reply:
Yep, an using an event is the right way to go. The way you are doing it you are hardcoding zombies into the anomaly system. If a fork does not have zombies this will break.

Reply:
Okay! I will make those changes tomorrow.

Reply:
Okay, @beck-thompson, do you have examples of where to look for how this is done?

Reply:
beck pointed at SelfBeforeGunShotEvent and other Before.*Event as examples. thanks!